docs/source/en/optimization/fp16.md

```diff
@@ -61,351 +57,6 @@ image = pipe(prompt).images[0]
 
 <Tip warning={true}>
 
-  It is strongly discouraged to make use of [`torch.autocast`](https://pytorch.org/docs/stable/amp.html#torch.autocast) in any of the pipelines as it can lead to black images and is always slower than using pure 
-  float16 precision.
+Don't use [`torch.autocast`](https://pytorch.org/docs/stable/amp.html#torch.autocast) in any of the pipelines as it can lead to black images and is always slower than pure float16 precision.
```

Comment:
This doc is one of our most visited docs - can we maybe make sure it looks to the newly created docs `memory.md` e.g. so that users have an easy time finding them?

Reply:
Added a `Tip` at the start of the `fp16.md` and `memory.md` doc that links to each other.